recipes/r-changepoints/meta.yaml

```
@@ -0,0 +1,100 @@
+{% set name = "r-changepoints" %}
+{% set version = "1.1.0" %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/changepoints_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/changepoints/changepoints_{{ version }}.tar.gz
+  sha256: 73b7e389d8d5590d91ecb369f57b80f605b3c9f231da04be54a650b9b9c593f1
+
+build:
+  merge_build_host: true  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}          # [not win]
+    - {{ compiler('cxx') }}        # [not win]
+    - {{ compiler('m2w64_c') }}    # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+
+  host:
+    - r-base
+    - r-gglasso
+    - r-glmnet
+    - r-ks
+    - r-mass
+    - r-data.tree
+    - r-rcpparmadillo
+    - liblapack        # [not win]
+    - libblas           # [not win]
+  run:
+    - r-base
+    - r-gglasso
+    - r-glmnet
+    - r-ks
+    - r-mass
+    - r-data.tree
+    - r-rcpparmadillo
+    - {{ native }}gcc-libs         # [win]
+
+
+test:
+  commands:
+    - $R -e "library('changepoints')"           # [not win]
+    - "\"%R%\" -e \"library('changepoints')\""  # [win]
+
+about:
+  home: https://github.com/HaotianXu/changepoints
+  summary: An R Package for Change Point Localisation
+  description: |
+    Performs a series of offline and/or online change-point localisation algorithms for
+
+    univariate mean
+        Wang, Yu and Rinaldo (2020)
+        Yu, Padilla, Wang and Rinaldo (2020)
+    univariate polynomials
+        Yu, Chatterjee and Xu (2021)
+    univariate and multivariate nonparametric settings
+        Padilla, Yu, Wang and Rinaldo (2021)
+        Padilla, Yu, Wang and Rinaldo (2021)
+    high-dimensional covariances
+        Wang, Yu and Rinaldo (2021)
+    high-dimensional networks with and without missing values
+        Wang, Yu and Rinaldo (2021)
+        Yu, Padilla, Wang and Rinaldo (2021)
+        Dubey, Xu and Yu (2021)
+    high-dimensional linear regression models
+        Rinaldo, Wang, Wen, Willett and Yu (2021)
+        Xu, Wang, Zhao, and Yu (2022)
+    high-dimensional vector autoregressive models
+        Wang, Yu, Rinaldo and Willett (2019)
+    high-dimensional self exciting point processes
+        Wang, Yu and Willett (2020)
+    dependent dynamic nonparametric random dot product graphs
+        Padilla, Yu and Priebe (2019)
+    robust univariate mean against adversarial attacks
+        Li and Yu (2021)
+  license: GPL-3.0-only
```

[License is GPL (>= 3.0)](https://github.com/HaotianXu/changepoints/blob/b96ed0c979ee155bb583d03a5023e9229801d8cf/DESCRIPTION#L30):

```suggestion
  license: GPL-3.0-or-later
```